truss-chains/truss_chains/model_skeleton.py

```diff
@@ -23,6 +24,24 @@ def __init__(
                 config["model_metadata"][definitions.TRUSS_CONFIG_CHAINS_KEY]
             )
         )
+
+        # Override chainlet_to_service if chainlet_service_config exists in extra config
+        chainlet_service_config = (
+            extra_config_resolver.ExtraConfigResolver.get_config_value(
+                "chainlet_service_config"
+            )
+        )
+        if chainlet_service_config:
+            chainlet_service_config_json = json.loads(chainlet_service_config)
+            for (
+                chainlet_name,
+                service_descriptor,
+            ) in truss_metadata.chainlet_to_service.items():
```

Comment:
Can we eliminate `truss_metadata.chainlet_to_service`?

Reply:
Synced with Marius - we'll keep the `chainlet_to_service` type around since we don't know what the `ServiceDescriptor.options` are when we create the `chainlet_url_map` in Django - this way we only use `chainlet_url_map` to override the `predict_url`